File: c/02-condicionais/main018.c
/*Faça um programa que leia o ano de nascimento de uma pessoa, calcule a idade
dela e depois mostre se ela pode ou não votar.

Data 30/09/2025 -> 9ª Dia*/


#include <stdio.h>

int main() {

    int ano;
    int idade;

    scanf("%d",&ano);

    idade = 2025 - ano;

    if(idade >= 18) {

        printf("Voce tem: %d anos de idade entao ja podes voltar",idade);
    }
    else {
        printf("Voce tem: %d anos de idade entao ainda nao podes votar",idade);
    }
    return 0;
}

File: python/02-condicionais/main018.py
'''Faça um programa que leia o ano de nascimento de uma pessoa, calcule a idade
dela e depois mostre se ela pode ou não votar.

Data 30/09/2025 -> 9ª Dia
'''

ano = int(input('Digite o seu ano de nascimento: '))
idade = 2025 - ano

if idade >= 18:
    print(f'Você tem: {idade} anos de idade então já tem idade para votar.')
else:
    print(f'Você tem: {idade} anos de idade então ainda podes votar')